components/src/web-components/visualization/gs-genome-data-viewer.stories.ts

```diff
@@ -0,0 +1,112 @@
+import { expect, waitFor } from '@storybook/test';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './gs-genome-data-viewer';
+import '../gs-app';
+import { withComponentDocs } from '../../../.storybook/ComponentDocsBlock';
+import { LAPIS_URL } from '../../constants';
+import type { GenomeDataViewerProps } from '../../preact/genomeViewer/genome-data-viewer';
+import { withinShadowRoot } from '../withinShadowRoot.story';
+
+const codeExample = String.raw`
+<gs-genome-data-viewer
+    gff3Source='https://raw.githubusercontent.com/nextstrain/nextclade_data/8f2e791d3a59013ee88e1d1d7e83b486d39c4ecb/data/nextstrain/wnv/all-lineages/genome_annotation.gff3'
+    genomeLength=11029
```

Comment:
I believe that you always need quotes in HTML, don't you?
```suggestion
    genomeLength="11029"
```